graphql/schema.graphql

```diff
@@ -608,18 +608,16 @@ input WorkstationSetupCommandInput {
 
 input PatchTriggerAliasInput {
   alias: String!
-  childProjectId: String!
   childProjectIdentifier: String!
-  taskSpecifiers: [TaskSpecifierInput]
+  taskSpecifiers: [TaskSpecifierInput!]!
   status: String
   parentAsModule: String
-  variantsTasks: [VariantTaskInput]!
 }
 
 input TaskSpecifierInput {
-  patchAlias: String!
-  taskRegex: String!
-  variantRegex: String!
+  patchAlias: String
+  taskRegex: String
+  variantRegex: String
```

Comment:
I figured that these were non-nullable because empty strings would be provided for the "inactive" choice. Using empty strings would be more in line with the approach for [ProjectAliasInput](https://github.com/evergreen-ci/evergreen/blob/86d205927187656bc70273447b308fcd51b54f63/graphql/schema.graphql#L636-L645), but either works.

Reply:
Okay that makes sense, i can change it back!